tests/e2e/tests/e2e/GitSsh.spec.ts

```diff
@@ -131,3 +141,10 @@ async function cloneTestRepo() {
     await quickOpenContainer.typeAndSelectSuggestion(sshLinkToRepo, confirmMessage);
     await gitPlugin.clickOnSelectRepositoryButton();
 }
+
+// workaround related to multi-root, issue: https://github.com/eclipse/che/issues/19101
```

Comment:
the actual issue for tracking now is #19544

Reply:
Thanks, replaced.